modules/core/db/update/hsql/18/181119-1-createDiagnoseExecutionLog.sql

```diff
@@ -0,0 +1,19 @@
+create table DDCRD_DIAGNOSE_EXECUTION_LOG (
```

Comment:
why is a table created here?

Reply:
Hi Mario,
looks like studio automatically created this update script